src/components/DashboardPage/DashboardPage.tsx

```diff
@@ -1,18 +1,32 @@
-import { Divider, PageSection, PageSectionVariants, TextContent, Text } from '@patternfly/react-core';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { IWebConfig } from '../../services/WebConfigService';
+import { PageLayout } from '../PageLayout/PageLayout';
+import { DashboardWidget } from '../DashboardWidget/DashboardWidget';
+import * as WebConfigAPI from '../../services/WebConfigService';
 
 export const DashboardPage = () => {
-  return (
-    <>
-      <PageSection variant={PageSectionVariants.light}>
-        <TextContent>
-          <Text component="h1">DashboardPage</Text>
-          <Text component="p">
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam quos unde, accusantium excepturi ad praesentium.
-          </Text>
-        </TextContent>
-      </PageSection>
+  const webConfigData = WebConfigAPI.getWebConfig() as IWebConfig;
```

Comment:
Hi @DnsZhou , I think that things like `as IWebConfig` or `as string` should not be needed anymore.

Reply:
I would also propose chaning `webConfigData` to `webConfig` so that it more reflects Web Config refactor changes.